### PR TITLE
Nr-309883 migrate from opensource newrelic

### PR DESCRIPTION
**Pull request by ubanerjeeNR**

### CODE REVIEW

File: src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: New Relic Open Source Community Code of Conduct - Reporting Guide
```
Comment (paperclypse) [inline src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx:2]: @ubanerjeeNR We use sentence case even in titles (https://docs.newrelic.com/docs/style-guide/capitalization/capitalization/#heading_caps)

I think this should be: "New Relic open source community code of conduct - reporting guide"

File: src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+If you are unsure whether the incident is a violation, or whether the space where it happened is covered by this [Code of Conduct](/docs/licenses/open-source/code-of-conduct), we encourage you to still report it. We would much rather have a few extra reports where we decide to take no action, rather than miss a report of an actual violation. We do not look negatively on you if we find the incident is not a violation. And knowing about incidents that are not violations, or that happen outside our spaces, can also help us to improve the [Code of Conduct](/docs/licenses/open-source/code-of-conduct) or the processes surrounding it.
+
+In your report please include information that will help us start an investigation, such as:
+
+- Your contact info (so we can get in touch with you if we need to follow up).
+- Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+- When and where the incident occurred. Please be as specific as possible.
+- Your account of what occurred. If there is a publicly available record (e.g., a mailing list archive or Slack history) please include a link.
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx:18]: But... license docs are where we make exceptions to a lot of these little rules :)

File: src/content/docs/licenses/open-source/code-of-conduct.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: New Relic Open Source Community Code of Conduct
+metaDescription: "Participate in New Relic's open source community with kindness and respect: our Code of Conduct sets the standard."
+redirects:
+  - https://opensource.newrelic.com/code-of-conduct
+freshnessValidatedDate: never
+---
+
+We believe in the power of Open Source to bring together developers and ideas to solve tricky problems and create the innovative software the world needs.
```
Comment (paperclypse) [inline src/content/docs/licenses/open-source/code-of-conduct.mdx:9]: "Open Source" should be "open source"

File: src/content/docs/licenses/open-source/oss-category.mdx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    - Issues and PR’s managed in GitHub
+    - For additional details on support options, see the [Open Source Support Policy](/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings?_gl=1*1ftd2ji*_gcl_aw*R0NMLjE3MjE3NDA0NzIuQ2p3S0NBandxZjIwQmhCd0Vpd0F0N2R0ZGQxa01LaHBmazl1OFNQZTlTRU02eHZ1UThaZXk3dVdoQ3F2Y09iX2JzOFJpbENRUDlwZVpob0NGdkFRQXZEX0J3RQ..*_gcl_au*MTUzNzk1ODMxOS4xNzIwNDI5NDM0*_ga*NzE1OTA0NDA1LjE3MjA0Mjk0MzU.*_ga_R5EF3MCG7B*MTcyNzQxNzgxOS4yMjIuMS4xNzI3NDE4OTQxLjYwLjEuMjI2NDAzOTc2)
+    - Documentation reviewed by the New Relic documentation team
+    - Linted code
+    - An automated release pipeline
+  </Collapser>
+
+  <Collapser
+    id="one-catalog"
+    title="New Relic One Catalog"
```
Comment (paperclypse) [inline src/content/docs/licenses/open-source/oss-category.mdx:56]: We don't use the brand "New Relic One" anymore. It should just be "New Relic catalog"

File: src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx (hunk 1 above)
Comment (ubanerjeeNR) [reply src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx:2]: fixed

File: src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+title: New Relic Open Source Community Code of Conduct - Reporting Guide
+metaDescription: Learn how to report violations of the New Relic code of conduct for open source community.
```
[the remark this answers is not in the document]
Comment (ubanerjeeNR) [reply src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx:3]: Okay

File: src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ubanerjeeNR) [reply src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx:18]: My bad - fixed it.

File: src/content/docs/licenses/open-source/code-of-conduct.mdx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+We respect all people in our community, regardless of their background or the extent of their contributions, and encourage participation from diverse sources. This Code of Conduct documents the principles we encourage in, and expect of, the Open Source community.
+
+- Be friendly and patient.
+- Be welcoming.<p/>We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to, members of any race, ethnicity, culture, national origin, colour, immigration status, social or economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
```
[the remark this answers is not in the document]
Comment (ubanerjeeNR) [reply src/content/docs/licenses/open-source/code-of-conduct.mdx:14]: Sure - why though?

File: src/content/docs/licenses/open-source/oss-category.mdx (hunk 4 above)
Comment (ubanerjeeNR) [reply src/content/docs/licenses/open-source/oss-category.mdx:56]: Got it

**Review by paperclypse — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**